scripts/nymms_api

```diff
@@ -10,4 +12,8 @@ logger = cli.setup_logging(args.verbose)
 config.load_config(args.config)
 
 from nymms.api import routes
+
+for plugin in config.settings.get('api', []).get('plugins', []):
```

Comment:
I almost wonder if it makes more sense to make the API plugin automatically get setup if the reactor handler is being used, because right now if you added that plugin but not the handler you'd run into issues, right?  I don't really know of a good way to handle this though.  May not be easy enough.